src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/request/NomisSyncRequest.kt

```diff
@@ -32,39 +30,3 @@ data class NomisSyncRequest(
     }
   }
 }
-
-@Schema(description = "Incident report created in NOMIS", accessMode = Schema.AccessMode.WRITE_ONLY)
-interface NomisSyncCreateRequest {
```

Comment:
interface not used anywhere?

Reply:
it was intended for clearer documentation. `NomisSyncRequest` was the form that the NOMIS migration side was using, but `NomisSyncCreateRequest` & `NomisSyncUpdateRequest` more accurately described the interaction between fields

Reply:
essentially, iff `id` was present, it's an update.
but an update does not make sense with `initialMigration`=true